### DISCUSSION TITLE
Shortcut to wrap visual block with parenthesis ?

### DISCUSSION THREAD

**Discussion by luyangliuable** [syl20bnr/spacemacs#15153] [Q&A] [answered]
Hi is there a quick shortcut in spacemacs using sexp to wrap a block of text with parenthesis? Also I do I disable automatic closed parenthesis? Thanks

**Comment by lebensterben** [comment, marked as answer]
in visual mode select the text, then in normal mode press s + (

**Reply by dalanicolai** [reply]
or `s + )`...
You can disable automatic closing of parentheses by disabling `smartparens-mode`.

**Reply by luyangliuable** [reply]
Thanks so much
